cmd/alertmanager/main.go

```diff
@@ -216,8 +220,8 @@ func main() {
 		inhibitor.Stop()
 		disp.Stop()
 
-		inhibitor = NewInhibitor(alerts, conf.InhibitRules, marker)
-		disp = NewDispatcher(alerts, NewRoute(conf.Route, nil), build(conf.Receivers), marker)
+		inhibitor = inhibit.NewInhibitor(alerts, conf.InhibitRules, marker)
```

Comment:
wdyt of applying the same scheme as for the `api` package here as in

```
inhibitor = inhibitor.New(...)
```

as the package doesn't contain anything but that..